### PR TITLE
fix(DB/SAI): Zulian Crocolisk has scripted movement

### PR DESCRIPTION
**Pull request by neifion-00000000**
-  Adds id 2 and 3 into creature 15043's smart scripts.
-  Movement loops 30 seconds of 0 movement, 10 seconds of random movement in a small radius

## SOURCE:

Sniffed timers and estimated movement values
## Tests Performed:

- Tested in ZG.  Reset looks a bit abrupt but needed to keep from wandering all over the instance.

## How to Test the Changes:

1. Goto ZG
2. Watch any croc pack for a couple mins

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by neifion-00000000** [comment]
special thanks @55Honey  for the radius suggestion, saved time over my first guess.

**Comment by ZhengPeiRu21** [comment]
Thank you Neifion! VMangos is having a field in their waypoint data table to set wander distance when a creature is at a waypoint with a long delay. If we see many more creatures that should be having movement patterns like this, I think we should implement that feature as well to save lots of manual scripting!

**Comment by neifion-00000000** [comment]
> Thank you Neifion! VMangos is having a field in their waypoint data table to set wander distance when a creature is at a waypoint with a long delay. If we see many more creatures that should be having movement patterns like this, I think we should implement that feature as well to save lots of manual scripting!

that won't impact this one but yes I agree, it would cut down on the what I feel is pointless need for SAI in a lot of other packs.

waypoint_data also has issues with emotes firing off in weird ways too so pretty much everything that isn't a strict path needs dealt with in SAI atm

**Comment by neifion-00000000** [comment]
found a bug in this, the mobs will now drift due to SAI random waypoint function

**Comment by neifion-00000000** [comment]
fixed, this requires #12468 to function

File: data/sql/updates/pending_db_world/rev_1658354450598559500.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--
+/* Crocs movement is a scripted action that occurs about every 30 seconds, not normal random movement */
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658354450598559500.sql:2]: no /* */ you sinner

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
image

**Review by acidmanifesto — APPROVED** [review]
Checks should be good. Too high on sugar booger to stay up for work flow to turn green.
**Snorting happy line noises**

**Comment by Gultask** [comment]
Tested and it really does look much more like retail 👍

File: data/sql/updates/pending_db_world/rev_1658354450598559500.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+--
+-- Crocs movement is a scripted action that occurs about every 30 seconds, not normal random movement
+UPDATE `creature` SET `wander_distance`=0, `MovementType`=0 WHERE `id1`=15043;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=15043;
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1658354450598559500.sql:5]: missing sourcetype, without it we risk deleting gameobject scripts for example

**Review by Nyeriah — CHANGES_REQUESTED** [review]

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**